python/kafkaopmon/OpMonFunction.py

```diff
@@ -0,0 +1,17 @@
+class OpMonFunction :
```

Comment:
Is there a reason why this was extracted into a separate file? 
Otherwise I would prefer to restore as it was. The reason being that the file was supposed to be unique to simplify the construction of the microservice: https://github.com/DUNE-DAQ/microservices/blob/5a8182e9459e7f451eff6aed4da4dd06ccdc2eb5/dockerfiles/microservices-dependencies.dockerfile#L44